bom/pom.xml

```diff
@@ -47,27 +47,22 @@
      -->
     <java-base-image.name>docker.io/eclipse-temurin:17-jre-focal</java-base-image.name>
     <jjwt.version>0.11.5</jjwt.version>
-    <!--
-      Explicitly override Kafka 3.1 client version defined by Quarkus because
-      vertx-kafka-client 4.2.x is incompatible with Kafka 3.1 client.
-      See https://github.com/quarkusio/quarkus/issues/22000
-    -->
-    <kafka-client.version>2.8.1</kafka-client.version>
-    <kafka.image.name>confluentinc/cp-kafka:7.2.0</kafka.image.name>
+    <kafka-client.version>3.2.1</kafka-client.version>
```

Comment:
do we still need to explicitly manage the kafka client version or can we simply rely on Quarkus picking the right one?

Reply:
The version referenced by Quarkus is `3.1.0`. I would like to use `3.2.x` here to have a version where [KAFKA-12256](https://issues.apache.org/jira/browse/KAFKA-12256) is fixed. We can remove the above line with the Quarkus update to `2.12.x` then.

Reply:
I see